src/views/insurance/Policy/Policy.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+  <div class="aig-container aig-view">
+    <transition name="fade" mode="out-in">
+      <Card class="policy-card" v-loading="loading" :element-loading-text="$t('general.loading')">
+        <div slot="body" class="policy-details-body">
+          <el-row class="header">
+              <router-link :to="policyListRoute" class="back-button">
+                <i class="back-icon el-icon-arrow-left"></i>
+              </router-link>
+
+              <h2>{{ $t('insurance.policy.androidBatteryInsurancePolicy') }}</h2>
+          </el-row>
+
+          <div class="content">
+            <PolicyInfo :data="policy" />
+            <DeviceInfo :data="deviceData" />
+            <ClaimInfo :data="claimProperties" />
+          </div>
+
+          <div class="footer">
+            <el-button
+              v-if="policy.status && policy.status.toUpperCase() === 'DRAFT'"
+              class="aig-button"
+              type="primary"
+              @click.prevent.native="insure">{{ $t('insurance.policy.insure') }}</el-button>
+
+            <el-button
+              v-else-if="policy.status && policy.status.toUpperCase() === 'PENDINGPAYMENT'"
+              class="aig-button"
+              disabled="true"
+              type="primary">{{ $t('insurance.policy.verifyForClaim') }}</el-button>
```

Comment:
@click event handler is missing ?

Reply:
This button is disabled if policy status is 'PENDINGPAYMENT' so in this case we do not need @click event here :)